README.asc

```diff
@@ -39,7 +39,7 @@ be sitting in front of a prompt:
 
 ----
 Connected to Test Cluster at localhost:9160.
-[cqlsh 2.2.0 | Cassandra 1.2.0 | CQL spec 3.0.0 | Thrift protocol 19.35.0]
+[cqlsh 6.0.0 | Cassandra 4.0.2 | CQL spec 3.4.5 | Native protocol v5]
```

Comment:
@bschoening this is irrelevant to this patch. Please revert it.

Reply:
We should probably bump the cqlsh version to 6.1.0, so I can put this update there.  But this is just a stale comment line fix, and potentially confusing with its reference to Thrift.

Reply:
@bschoening want to file a new ticket for this?